src/pages/InstitutionsPage.jsx

```diff
@@ -101,48 +114,49 @@ function InstitutionsPage(props) {
 
     return (
         <div className="d-flex flex-column vh-100">
-            <div className="row h-100 m-0">
-                <div className="col-auto bg-coral-red d-flex position-lg-sticky h-100 top-0 p-0">
+            <div className="row align-items-stretch h-100 g-0">
+                <div className="col-auto d-flex bg-coral-red position-lg-sticky top-0">
                     <div className="offcanvas-lg offcanvas-start bg-coral-red d-flex w-auto" tabIndex="-1" id="sidebar">
                         <Sidebar showExitButton={false} />
                     </div>
                 </div>
-                <div className="col d-flex flex-column h-100 p-0">
+                <div className="col d-flex flex-column h-100">
                     <NavBar showNavTogglerMobile={true} showNavTogglerDesktop={false} />
-                    <div className="row align-items-center justify-content-center font-barlow m-0">
-                        <div className="col-12 col-md-10 p-4 pb-0">
-                            <h1 className="color-grey font-century-gothic fw-bold fs-2 m-0">Instituições</h1>
-                        </div>
-                    </div>
-                    <div className="row justify-content-center font-barlow flex-grow-1 m-0 overflow-hidden">
-                        <div className="col col-md-10 d-flex flex-column h-100 p-4">
-                            <h1 className="color-grey font-century-gothic text-nowrap fw-bold fs-3 pb-4 m-0">Minhas instituições</h1>
-                            <div className="d-flex justify-content-center flex-grow-1 overflow-hidden">
-                                <ProtocolList
-                                    listItems={visibleInstitutions.map((i) => ({ id: i.id, title: i.name }))}
-                                    hsl={[36, 98, 83]}
-                                    allowEdit={user.role === 'ADMIN' || user.role === 'COORDINATOR'}
-                                    allowDelete={user.role === 'ADMIN'}
-                                    viewFunction={(id) => navigate(`${id}`)}
-                                    editFunction={(id) => navigate(`${id}/manage`)}
-                                    deleteFunction={(id) => deleteInstitution(id)}
-                                />
+                    <div className="d-flex flex-column flex-grow-1 overflow-y-auto p-4">
+                        <CustomContainer
+                            className="font-barlow flex-grow-1 overflow-lg-y-hidden"
+                            childrenClassName="mh-100"
+                            df="12"
+                            md="10"
+                        >
+                            <h1 className="color-grey font-century-gothic fw-bold fs-2 mb-4">Instituições</h1>
+                            <div class="row flex-grow-1 overflow-lg-y-hidden pb-lg-4 g-4">
```

Comment:
```suggestion
                            <div className="row flex-grow-1 overflow-lg-y-hidden pb-lg-4 g-4">
```

Esse overflow sem "lg" não funciona em telas menores que o breakpoint do lg, alguma razão para tal?